### ISSUE TITLE
Development: `npm run start` requires an initial reload

### ISSUE THREAD

**Issue by bryphe** [onivim/oni#1345]
To iterate quickly, we use `npm run start` which creates several 'watchers', spins up a second instance of Oni, and reloads that instance whenever source files change - live. This is great for seeing the result of changes quickly.

However, there is a bug on Linux/OSX - on the first load, the developer needs to _explicitly_ reload the page once compiliation finishes, ie `window.location.reload()` in the console.

It'd be better if the logic in our `index.html` is more robust - we have this code here to pick up the webpack-dev-server bundle:
```
    if(process.env.ONI_WEBPACK_LOAD) {
        path = "http://localhost:8191/bundle.js";
        const hotReloadElement = document.createElement("div");
        hotReloadElement.textContent = "Waiting for webpack compilation... You may need to reload the browser once webpack finishes compiling."
        document.body.appendChild(hotReloadElement)
    }
```

But it only tries once to load the bundle. It'd be better if we retried or listened for the load to be complete, and then reload. The result should be that we need no additional interaction past running `npm run start` to start seeing live changes.

**Comment by someguynamedmatt** [comment]
@TalAmuyal I can't recreate this. `npm run start` seems to work as expected since @bryphe merged #1368

**Comment by someguynamedmatt** [comment]
Nevermind. I just recreated the behavior.

**Comment by bryphe** [comment]
Should be addressed by #1663 now - let me know if you still see issues!
